.gitignore

```diff
@@ -0,0 +1,79 @@
+
+# Created by https://www.gitignore.io/api/go,jetbrains
+
+### Go ###
+# Compiled Object files, Static and Dynamic libs (Shared Objects)
+*.o
+*.a
+*.so
+
+# Folders
+_obj
+_test
+
+# Architecture specific extensions/prefixes
+*.[568vq]
+[568vq].out
+
+*.cgo1.go
+*.cgo2.c
+_cgo_defun.c
+_cgo_gotypes.go
+_cgo_export.*
+
+_testmain.go
+
+*.exe
+*.test
+*.prof
+
+
+### JetBrains ###
```

Comment:
I'm tempted to say that editor specific stuff should be part of your local gitignore, rather than the repo, but meh